sbom.json

```diff
@@ -0,0 +1,51 @@
+{
+  "metadata": {
+    "timestamp": "2024-06-10T22:49:06.475581+00:00",
+    "tools": [
+      {
+        "externalReferences": [
+          {
+            "type": "build-system",
+            "url": "https://github.com/CycloneDX/cyclonedx-python-lib/actions"
+          },
+          {
+            "type": "distribution",
+            "url": "https://pypi.org/project/cyclonedx-python-lib/"
+          },
+          {
+            "type": "documentation",
+            "url": "https://cyclonedx-python-library.readthedocs.io/"
+          },
+          {
+            "type": "issue-tracker",
+            "url": "https://github.com/CycloneDX/cyclonedx-python-lib/issues"
+          },
+          {
+            "type": "license",
+            "url": "https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/LICENSE"
+          },
+          {
+            "type": "release-notes",
+            "url": "https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/CHANGELOG.md"
+          },
+          {
+            "type": "vcs",
+            "url": "https://github.com/CycloneDX/cyclonedx-python-lib"
+          },
+          {
+            "type": "website",
+            "url": "https://github.com/CycloneDX/cyclonedx-python-lib/#readme"
+          }
+        ],
+        "name": "cyclonedx-python-lib",
```

Comment:
Please remove this from dependencies list as we do not use this tool.

Reply:
Done. It is beneficial to remove the metadata section of the SBOM to prevent confusion. The tool in question is listed as part of that metadata section to show which tools were used to produce the SBOM. It is understandable how this confusion can arise, so we should simplify the SBOM by removing the metadata. We need to make it clear to anyone looking at this SBOM in the repo that the tool is not one of our dependencies.